src/views/App/App.jsx

```diff
@@ -7,24 +7,29 @@ import Dashboard from "../Dashboard";
 import Profile from "../Profile";
 import Sidebar from "../../components/Sidebar";
 
-import { BrowserRouter, Route, Switch } from "react-router-dom";
+import {BrowserRouter, Route, Switch} from "react-router-dom";
 import BillSnapBackground from "../Background";
 import "./styles.scss";
 
 /**
  * Load you page components here. We will render them via routes
  */
 
-export default () => {
+export default (props) => {
+
+  if(localStorage.getItem("billSnap_token")){
```

Comment:
If there's a better spot for this little nugget, then lmk

Reply:
I'm thinking, we'll only want to call this once; we should put it in a `useEffect` function and treat it as a `componentDidMount`

see [example](https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup)